src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor());
+        ball.setNumber(new Random().nextInt(101));
+        return ball.toString();
+    }
```

Comment:
Instead of creating a new instance of ColorSupplier and Random every time the method is called, it would be more efficient to have them as private final fields of the Lottery class and reuse the same instances.